components/Navigation/Navigation.js

```diff
@@ -0,0 +1,36 @@
+import Image from "next/image";
+import closeIcon from "@/public/closeIcon.svg";
+import {
+  StyledNav,
+  StyledNavWrapper,
+  StyledUl,
+  StyledNavLink,
+  StyledCloseIcon,
+} from "./Navigation.styled";
+
+const Navigation = ({ active }) => {
+  const closeSidebar = () => {
+    active(false);
+  };
+  return (
+    <StyledNavWrapper>
+      <StyledCloseIcon>
+        <Image
+          src={closeIcon}
+          width={30}
+          height={30}
+          alt="close icon"
+          onClick={closeSidebar}
+        />
+      </StyledCloseIcon>
```

Comment:
If you choose to implement the behavior outside the component, this will change to:
```suggestion
const Navigation = ({ onClose }) => {
  return (
    <StyledNavWrapper>
      <StyledCloseIcon>
        <Image
          src={closeIcon}
          width={30}
          height={30}
          alt="close icon"
          onClick={onClose}
        />
      </StyledCloseIcon>
```

Reply:
Thank you Ernst! I implemented the suggestions!